BlueprintUICommonControls/Sources/AccessibilityFocus.swift

```diff
@@ -0,0 +1,117 @@
+import BlueprintUI
+import UIKit
+
+/// Enables VoiceOver focus to jump to the wrapped element
+/// either on appearance or via a trigger that can be manually fired.
+public struct AccessibilityFocus: Element {
```

Comment:
Missing a public init

Reply:
(Unless you only want to allow this to be used via the `Element` builder – in which case, ignore me!)

Reply:
also fwiw a common pattern is often to call the parameter `wrapping` but the property `wrapped`, like so:
```swift
struct Foo: Element {
  var wrapped: Element
  init(wrapping wrapped: Element) {
    self.wrapped = wrapped
  }
}